packages/mdc-snackbar/_mixins.scss

```diff
@@ -0,0 +1,94 @@
+//
+// Copyright 2018 Google Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+
+@import "@material/elevation/mixins";
+@import "@material/shape/mixins";
+@import "@material/theme/functions";
+@import "@material/theme/mixins";
+@import "@material/theme/variables"; // for mdc-theme-prop-value() and mdc-theme-text-emphasis()
+@import "./variables";
+
+@mixin mdc-snackbar-fill-color($color) {
+  .mdc-snackbar__surface {
+    @include mdc-theme-prop(background-color, $color);
+  }
+}
+
+@mixin mdc-snackbar-label-ink-color($color) {
+  .mdc-snackbar__label {
+    @include mdc-theme-prop(color, $color);
+  }
+}
+
+@mixin mdc-snackbar-shape-radius($radius, $rtl-reflexive: false) {
+  .mdc-snackbar__surface {
+    @include mdc-shape-radius($radius, $rtl-reflexive: false);
```

Comment:
I assume this is a typo and we should simply be passing `$rtl-reflexive` through?

Reply:
Yes, good catch!